modules/Makefile

```diff
@@ -0,0 +1,91 @@
+# requires AWS_PROFILE
+# bucket must exist, prefix will be created
+S3_BUCKET ?= "s4c-cft"
+S3_PREFIX ?= "test"
+S3_REGION ?= eu-west-1
+STACK_NAME = Sysdig-Secure
+PARAM_NAME_SUFFIX ?= test
+PARAM_IS_ORGANIZATIONAL ?= false
+PARAM_EXTERNAL_ID ?= test
+PARAM_TRUSTED_IDENTITY ?= arn:aws:iam::064689838359:role/$(PARAM_NAME_SUFFIX)
```

Comment:
Is it okay to have this accountID hardcoded?

Reply:
for scanning yes, but for this, no.  good catch.

Reply:
removed